UIforETW/ChildProcess.cpp

```diff
@@ -29,7 +28,7 @@ ChildProcess::ChildProcess(std::wstring exePath)
 	// Create the pipe here so that it is guaranteed to be created before
 	// we try starting the process.
 	hPipe_ = CreateNamedPipe(kPipeName,
-		PIPE_ACCESS_DUPLEX | PIPE_TYPE_BYTE | PIPE_READMODE_BYTE,
+		(PIPE_ACCESS_DUPLEX | PIPE_TYPE_BYTE | PIPE_READMODE_BYTE),

```

Comment:
Why the parentheses? They're already on a separate line and separated by commas from other arguments. I guess I don't care either way, but it seems redundant.